spec/models/stripe_wrapper_spec.rb

```diff
@@ -1,46 +1,68 @@
 require 'rails_helper'
 
 describe StripeWrapper do
+  let(:valid_token) do
+    Stripe::Token.create(
+      :card => {
+          :number => "4242424242424242",
+          :exp_month => 10,
+          :exp_year => 2016,
+          :cvc => "314"
+        },
+      ).id
```

Comment:
Lines 11 and 12 are indented a bit more than they should. Hash `}` should be at the key level. `)` should be at the caller level (Stripe::Token.create) in this case.